<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>682</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>QLibraryLogin</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resources/images/LibraryLogo.png</normaloff>:/resources/images/LibraryLogo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="12,5">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/resources/images/LibraryBackground.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="loginWidget" native="true">
     <property name="font">
      <font>
       <family>Roboto</family>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="logoLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Montserrat</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"> image: url(:/resources/images/LibraryLogo.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="10,3">
        <item row="0" column="0">
         <widget class="QLabel" name="hostLabel">
          <property name="font">
           <font>
            <family>Montserrat</family>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Host</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="hostLineEdit">
          <property name="font">
           <font>
            <family>Montserrat</family>
           </font>
          </property>
          <property name="maxLength">
           <number>80</number>
          </property>
          <property name="placeholderText">
           <string>localhost</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="portLineEdit">
          <property name="font">
           <font>
            <family>Montserrat</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="maxLength">
           <number>80</number>
          </property>
          <property name="placeholderText">
           <string>3306</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="portLabel">
          <property name="font">
           <font>
            <family>Montserrat</family>
           </font>
          </property>
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="userNameLabel">
        <property name="font">
         <font>
          <family>Montserrat</family>
         </font>
        </property>
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userNameLineEdit">
        <property name="font">
         <font>
          <family>Montserrat</family>
         </font>
        </property>
        <property name="maxLength">
         <number>80</number>
        </property>
        <property name="placeholderText">
         <string>username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="passwordLabel">
        <property name="font">
         <font>
          <family>Montserrat</family>
         </font>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="font">
         <font>
          <family>Montserrat</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>80</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>password</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="rememberMeCheckBox">
        <property name="font">
         <font>
          <family>Montserrat</family>
         </font>
        </property>
        <property name="text">
         <string>Remember me</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="loginButton">
        <property name="font">
         <font>
          <family>Montserrat</family>
         </font>
        </property>
        <property name="text">
         <string>Log In</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="retrievePasswordButton">
        <property name="font">
         <font>
          <family>Montserrat</family>
         </font>
        </property>
        <property name="text">
         <string>Retrieve Password</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
